python-packages/order_utils/src/zero_ex/json_schemas/__init__.py

```diff
@@ -8,6 +8,51 @@
 import jsonschema
 
 
+class LocalRefResolver(jsonschema.RefResolver):
+    """Resolve package-local JSON schema id's."""
+
+    def __init__(self):
+        """Initialize a new LocalRefResolver instance."""
+        self.ref_to_file = {
+            "/addressSchema": "address_schema.json",
+            "/hexSchema": "hex_schema.json",
+            "/orderSchema": "order_schema.json",
+            "/wholeNumberSchema": "whole_number_schema.json",
+            "/ECSignature": "ec_signature_schema.json",
+            "/signedOrderSchema": "signed_order_schema.json",
+            "/ecSignatureParameterSchema": (
+                "ec_signature_parameter_schema.json" + ""
```

Comment:
While you're in here would you mind removing this redundant string concatenation and the parenthesis and putting this all on one line?

Reply:
I have a feeling it was added there by `black` because it passes the max char length? I will take a look and make the change (or let you know)

Reply:
I confirm this is black